lib/elixir/lib/option_parser.ex

```diff
@@ -173,18 +173,18 @@ defmodule OptionParser do
       {[debug: true], ["path/to/file"]}
 
       iex> OptionParser.parse!(["--limit", "xyz"], strict: [limit: :integer])
-      ** (OptionParser.ParseError) 1 error found!
-      --limit : Expected type integer, got "xyz"
+      ** (OptionParser.ParseError) found errors:
+      --limit - expected a value of type integer, got: "xyz"
```

Comment:
Oh, that's where it came from D: Any chance we can change at least the capital letter, as in `--limit : expected ...`? I mean in the Elixir command line as well. There may be reasons we do it like this but I don't know them, so pardon my ignorance here 😄